python/build-wheel-manylinux1.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+cat << EOF > "/usr/bin/nproc"
+#!/bin/bash
+echo 1
+EOF
+chmod +x /usr/bin/nproc
+
+mkdir .whl
+for PYTHON in cp27-cp27mu cp33-cp33m cp34-cp34m cp35-cp35m cp36-cp36m; do
+  rm -rf *
+  git checkout *
+  pushd python
+  /opt/python/${PYTHON}/bin/pip install numpy
```

Comment:
Thanks for the tip, I fixed this as well! I tried building the wheel with a new version of numpy and then running with an old version of numpy and wasn't able to cause the problem just now however.